src/server/auth/server/testing/auth_integration_test.go

```diff
@@ -32,6 +32,11 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/pps"
 )
 
+// works with global.postgresql.postgresqlAuthType: md5, which is default in values file.
+var default_test_opts = minikubetestenv.WithValueOverrides(map[string]string{
```

Comment:
Idiomatic Go doesn't use `snake_case` variable names.  Name this defaultTestOptions.  (Abbreviation is also of unclear value, unless the type name is Opt or Opts, maybe.  But it's "Options" here.)

Reply:
woops, I was thinking in the wrong language. fixed.